curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-068.md

```diff
@@ -1,22 +1,34 @@
 ---
-id: 5f3ef6e01f288a026d709587
+id: 5f3f26fa39591db45e5cd7a0
 title: Part 68
 challengeType: 0
 dashedName: part-68
 ---
 
 # --description--
 
-You can use an `hr` element to display a divider between sections of different content.
+The default properties of an `hr` element will make it appear as a thin light grey line. You can change the height of the line by specifying a value for the `height` property.
 
-First, add an `hr` element between the first `header` element and the `main` element.
+Change the height the `hr` element to be `3px`.
 
 # --hints--
 
-Test 1
+You should use the `hr` selector.
 
 ```js
+assert(code.match(/hr\s*{/i));
+```
+
+You should set the `height` property to `3px`.
 
+```js
+assert(code.match(/height:\s*3px;?/i));
+```
+
+Your `hr` element should have a height of `3px`.
+
+```js
+assert($('hr').css('height') === '3px');
```

Comment:
`hr {height: 3px;}` does not seem to pass.

Reply:
This passes on my end - what browser are you using?